README.md

```diff
@@ -44,8 +44,7 @@ The following files and folders enable our continuous integration system.
    ```
    rosdep install --from-path src --ignore-src -y
    ```
-    
-4. You should now be able to build the project with `catkin_make`.
+5. You should now be able to build the project with `catkin_make`.
```

Comment:
Why did this get changed?

Reply:
There was a line with install instructions for h5py/keras in the readme that I put in. Then I took it out when I added the dependencies to the package.xml. I'll add it back in since rosdep doesn't yet have the packages.